www/js/search.js

```diff
@@ -1111,25 +1126,24 @@ module.exports = {
 
   clickShabad(e, ShabadID, LineID, Line, rows, mode = 'click') {
     if (window.socket !== undefined) {
-      let shabadIdsplit = [ShabadID];
-      if (typeof ShabadID === 'string') {
-        shabadIdsplit = ShabadID.split('-');
+      let sessionKeySplit;
```

Comment:
what is this used for?

Reply:
In this PR I changed name from shabadIdsplit to sessionKeySplit because that made more sense. As ceremonyID or baniID is not shabadID. 

However, in general a session Key is the ShabadID + session type. So for Ceremony it would be ceremony-ceremonyID, for bani it would be bani-baniLength-baniID.I have already  marked this code for improvement. Need better / simpler way to differentiate between 

a) Different type of shabads / verses (One routine to generally show shabad, and then some inherited routines to show ceremony, shabad, bani etc separately)
b) Different state of when they are loaded (search / from history etc). (Probably using one state management object rather than using bunch of global state variables) 

Currently all that is done by passing arguments which makes the whole logic little complex.

Reply:
Created a ticket for this improvement.